tests/foreman/api/test_smartproxy.py

```diff
@@ -78,7 +78,7 @@ def test_positive_create_with_name(self):
             with self.subTest(name):
                 new_port = get_available_capsule_port()
                 with default_url_on_new_port(9090, new_port) as url:
-                    proxy = self._create_smart_proxy(name=name, url=url)
+                    proxy = self._create_smart_proxy(name=name, url=url[0])
```

Comment:
this was a little intentional from my side as i was considering adding the channel.send() inside :) but yeah, will update it if i won't use it after all